src/core/tests.py

```diff
@@ -1422,3 +1448,47 @@ def test_email(self):
         )
         self.assertEqual(resp.status_code, 200)
         self.assertEqual("was sent" in resp.content, True)
+
+    def test_book_get_all_editors(self):
+        press_editor = UserFactory(username='preston')
+        series_editor = UserFactory(username='saul')
+        book_editor = UserFactory(username='edmund')
+
+        series = SeriesFactory(
+            name='Global Guides to Testing',
+            editor=series_editor
+        )
+        book = BookFactory(title='Transatlantic Testing Networks',)
+
+        editors = book.get_all_editors()
+        self.assertEqual(len(editors), 0)
+
```

Comment:
There are a lot of tests going on here in this one test. Would it not be a good idea to split this into several different tests, one for each assert?